<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DElegirUsuarios</class>
 <widget class="QDialog" name="DElegirUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="leId_Elegir">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lCorreo_Elegir">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce el correo del usuario que quieras modificar:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>96</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>96</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
